src/doc/trpl/the-stack-and-the-heap.md

```diff
@@ -1,3 +1,570 @@
 % The Stack and the Heap
 
-Coming Soon
+As a systems language, Rust operates at a low level. If you’re coming from a
+high-level language, there are some aspects of systems programming that you may
+not be familiar with. The most important one is how memory works, with a stack
+and a heap. If you’re familiar with the terms ‘stack’ and ‘heap’ with regards
+to memory management, this chapter will be a refresher. If you’re not, you’ll
```

Comment:
perhaps change to "if you're familiar with how C-like languages use stack allocation, ...", per the feedback